packages/dashboard-core-plugins/src/ChartPlugin.tsx

```diff
@@ -11,16 +11,13 @@ import shortid from 'shortid';
 import { ChartPanel } from './panels';
 import { ChartEvent } from './events';
 
-export type ChartPluginComponentProps = DashboardPluginComponentProps & {
+export type ChartPluginProps = Partial<DashboardPluginComponentProps> & {
   hydrate: PanelHydrateFunction;
 };
 
-export const ChartPlugin = ({
-  id,
-  layout,
-  registerComponent,
-  hydrate = DashboardUtils.hydrate,
-}: ChartPluginComponentProps): JSX.Element => {
+export const ChartPlugin = (props: ChartPluginProps): JSX.Element => {
+  assertIsDashboardPluginProps(props);
+  const { id, layout, registerComponent, hydrate } = props;
```

Comment:
In the other files you have hydrate w/ a default value. Just make it consistent w/ the others